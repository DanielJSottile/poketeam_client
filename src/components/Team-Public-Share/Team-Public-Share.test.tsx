import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
//import { render } from '@testing-library/react';
import TeamPublicShare from './Team-Public-Share';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<TeamPublicShare>', (): void => {
  it('renders without crashing', (): void => {
    const div: HTMLDivElement = document.createElement('div');
    const team = {team_name: 'test'}
    ReactDOM.render(
      <BrowserRouter>
        <TeamPublicShare team={team}/>
      </BrowserRouter>
      , div);
      ReactDOM.unmountComponentAtNode(div);
  });
  it('snapshot', (): void => {
    const team = {team_name: 'test'}
    const wrapper = shallow(<TeamPublicShare team={team}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});