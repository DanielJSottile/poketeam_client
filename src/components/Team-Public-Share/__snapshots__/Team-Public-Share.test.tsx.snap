// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamPublicShare> snapshot 1`] = `
<Fragment>
  <section
    className="team-section"
    id="undefined"
  >
    <div
      className="team-closed"
      onClick={[Function]}
    >
      <div>
        <h3>
          test
        </h3>
      </div>
      <div>
        <p>
          By 
        </p>
        <p>
          Created on:
           
          Invalid Date
        </p>
      </div>
    </div>
  </section>
</Fragment>
`;
